eng/pipelines/coreclr/templates/build-perf-sample-apps.yml

```diff
@@ -165,3 +166,133 @@ steps:
         artifactName: iOSSampleAppNoSymbols
         archiveExtension: '.zip'
         archiveType: zip
+
+  - ${{ if and(eq(parameters.osGroup, 'ios'), eq(parameters.nameSuffix, 'iOSMono'), eq(parameters.hybridGlobalization, True)) }}:
+    - script: make build-appbundle TARGET=ios MONO_ARCH=arm64 MONO_CONFIG=Release AOT=True USE_LLVM=False DEPLOY_AND_RUN=false STRIP_DEBUG_SYMBOLS=false HYBRID_GLOBALIZATION=true
```

Comment:
You can pass the parameters.hybridGlobalization to the command and avoid this branch. In that case, you would also need to update the artifactName according to the parameter value.

Reply:
Great catch.